hoggar/prisma/migrations/20221105155546_initial/migration.sql

```diff
@@ -0,0 +1,53 @@
+-- CreateTable
+CREATE TABLE "Subscription" (
+    "id" TEXT NOT NULL PRIMARY KEY,
+    "createdAt" DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" DATETIME NOT NULL,
+    "email" TEXT NOT NULL,
+    "source" TEXT
+);
+
+-- CreateTable
+CREATE TABLE "Identity" (
+    "id" TEXT NOT NULL PRIMARY KEY,
+    "createdAt" DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" DATETIME NOT NULL,
+    "provider" TEXT NOT NULL,
+    "providerId" TEXT NOT NULL,
+    "email" TEXT NOT NULL
+);
+
+-- CreateTable
+CREATE TABLE "Session" (
+    "id" TEXT NOT NULL PRIMARY KEY,
+    "createdAt" DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" DATETIME NOT NULL,
+    "uaIdentifier" TEXT NOT NULL,
+    "identityId" TEXT NOT NULL,
+    CONSTRAINT "Session_identityId_fkey" FOREIGN KEY ("identityId") REFERENCES "Identity" ("id") ON DELETE CASCADE ON UPDATE CASCADE
+);
+
+-- CreateTable
+CREATE TABLE "Profile" (
+    "id" TEXT NOT NULL PRIMARY KEY,
+    "createdAt" DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" DATETIME NOT NULL,
+    "identityId" TEXT NOT NULL,
+    "name" TEXT NOT NULL,
+    "bio" TEXT NOT NULL DEFAULT '',
+    "githubUser" TEXT NOT NULL DEFAULT '',
+    "avatarUrl" TEXT NOT NULL DEFAULT '',
+    "websiteUrl" TEXT NOT NULL DEFAULT '',
+    "location" TEXT NOT NULL DEFAULT '',
+    "company" TEXT NOT NULL DEFAULT '',
+    CONSTRAINT "Profile_identityId_fkey" FOREIGN KEY ("identityId") REFERENCES "Identity" ("id") ON DELETE CASCADE ON UPDATE CASCADE
+);
+
+-- CreateIndex
```

Comment:
Still need index on identity (provider, providerId)

Reply:
i see the code i think there is no need to index betwen (provider , providerId)